arangosh/Shell/ClientFeature.cpp

```diff
@@ -388,7 +389,7 @@ std::unique_ptr<httpclient::SimpleHttpClient> ClientFeature::createHttpClient(
 }
 
 std::vector<std::string> ClientFeature::httpEndpoints() {
```

Comment:
It seems that this method only cares about endpoint 0, which may or may not be right.
However, I can't find any caller of this method, so what about circumventing this problem and simply removing the method completely?

Reply:
Addressed in commit